.gitignore

```diff
@@ -24,3 +24,8 @@ conda-build-testing
 rever/
 **/.vscode
 **/_build
+
+# Remove all the `.bat`s and `.sh`s files the recipie folder creates
+
+/test-recipie/*.*/*.bat
+/test-recipie/*.*/*.sh
```

Comment:
These patterns are wrong and won't match anything.

I also don't think they're desired since we have `.bat` & `.sh` scripts in those locations that we do want to keep.

Reply:
the recipe folders make the .bat and .sh so this is prevent the upload of the files..

Reply:
_some_ of the tests auto-generate their own .bat/.sh files, other tests define their own custom .bat/.sh files, this global pattern is too aggressive to cover both of these cases